open-machine-learning-jupyter-book/assignments/llm/basic/transformer-architecture.ipynb

```diff
@@ -0,0 +1,826 @@
+{
```

Comment:
<p>I think this can be placed before the code block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>